+dj/Table.m

```diff
@@ -215,109 +199,98 @@ function erd(self, up, down)
                 up = up(1);
             end
             
-            self.schema.conn.erd({self.fullTableName},up,down)
+            d = dj.ERD(self);
+            n = length(d.nodes);
+            while down>0 || up>0
+                if down>0
+                    down = down - 1;
+                    d.down
+                end
+                if up>0
+                    up = up - 1;
+                    d.up
+                end
+                if length(d.nodes)==n
+                    break
+                end
+                n = length(d.nodes);
+            end
+            d.draw
         end
         
         
         function str = re(self)
-            % dj.Table/re - "reverse engineer" the table declaration.
+            % alias for self.re
+            warning 'dj.Table/re is deprecated: Use dj.Table/describe instead'
```

Comment:
Depends on whether we care to make this a withstanding feature or note. I'm quite certain that if we let it linger around, someone will use it and thus we'll have to continue to support it. I think this is a good time for us to decide whether to remove it or keep it. If we are keeping it now, we'll keep it as an officially supported feature. I'm fine either way but felt that provision of prefix can complicate things unnecessarily.